wiki/Rules/Visual_Content_Considerations/en.md

```diff
@@ -1,117 +1,52 @@
+## Criteria
 
-### Romance, flirting or intimate affection
+Images or visual elements containing any of the following are **not allowed**:
 
-Depiction of romantic acts or common intimacy between characters in an established narrative context is acceptable so long as it does not tread upon the rules listed above.
+- **sexual content involving or targeted at minors**
+- **excessive violence, gore, dismemberment, decapitation, or maiming**
+- **significant sexual innuendo**
+- **sexual posturing or solicitation**
+- **erotic content or graphic displays of sexuality**
+- **examples of drug use, preparation, or identification with illicit drug subculture**
+- **depictions of deliberately inflammatory political, cultural, religious, or social content**
 
-This also includes "flirty" depictions which are - in good faith - considered as playful, not unreasonably excessive, and not in violation of any of the other inappropriate content markers.
+## Exceptions
 
-## Further considerations
+Images or visual elements containing examples of the following may be considered acceptable:
 
-The anecdotal "rule of thumb" is that one should generally be comfortable with their family members or other judgemental authority figure viewing the content and be capable of adequately explaining what is going on in it.
+- mild to moderate depictions of violence (small amounts of blood, non-critical or superficial wounding)
+- artistic nudity (in the context of classical artwork, or work related to the element, such as album art with minor nudity present)
+- mild romance, flirting or intimate affection (hugging, embracing, kissing without sexual connotations, hand-holding, etc.)
+- swimsuits of reasonable design (standard swimwear with general coverage, ordinary bikini)
 
-This is by no means a strict requirement, but it is a good way to help inform your decisions on what is and isn't appropriate.
+## Getting your image assessed
 
-### What do I do if I am uncertain about a piece of visual content?
+Images that are used anywhere on the site outside of a beatmap (avatars, profile covers, userpages, etc) are **not** subject to the voting procedure explained below and may be removed at any time at the discretion of the Global Moderation Team.
 
-In situations whether the appropriateness of a given piece of visual content is unclear, the mapper must submit a formal request to any Beatmap Nominator for further appraisal, who will then refer the issue to the Global Moderation Team to decide based on a supermajority (>70%) consensus.
+If you are unsure about an image that is going to be included in a beatmap in any way, **please submit a formal request to any [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators), member of the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) or member of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).**
```

Comment:
can this be replaced with at very least a google form or something, which posts to discord? seems pretty annoying to have people reaching out in private to moderators.

Reply:
talking with pishi now about getting it added to existing BN site report functionality which is much less prone to abuse